consts.go

```diff
@@ -0,0 +1,24 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
```

Comment:
Oops. I changed it somewhere and it's not one of the files in this PR, so I wonder which of my files I changed...